common/config/rush/pnpm-lock.yaml

```diff
@@ -946,7 +947,7 @@ packages:
   /@types/body-parser/1.19.0:
     dependencies:
       '@types/connect': 3.4.34
-      '@types/node': 8.10.66
+      '@types/node': 10.17.54
```

Comment:
@xirzec

Reply:
does `rush --recheck` do the right thing? (i.e. leave it at 8.x)

Reply:
@deyaaeldeen can you revert the current lock file and then try the recheck flag?

Reply:
I tried `rm common/temp/pnpm-lock.yaml && rush update --purge --recheck` and it resulted in the exact same changes.

Reply:
/cc @mikeharder @praveenkuttappan in case their expertise can help with this weird issue

Reply:
I believe the version of `@types/node` in `pnpm-lock.yaml` flips between v8 and v10 frequently, but we never figured out why and just ignore it.

Reply:
We have @types/node ^10.0.0 explicitly allowed in common-versions.json. A couple of packages list that as their types/node dependency version, and these packages that get bumped up to 10.x in the lock file probably have a wildcard version spec in their dependencies or something that lets them move up.

Reply:
After I merged in master, the regression went away! just did a normal `rush update`.